cycledash/genotypes.py

```diff
@@ -92,16 +92,16 @@ def get(vcf_id, query, with_stats=True):
      limit: 250
     }
     """
-    query = _annotate_query_with_types(query, spec(vcf_id))
-    compare_to_vcf_id = query.get('compareToVcfId')
+    query = _annotate_query_with_types(query, spec(run_id))
+    compare_to_run_id = query.get('compareToVcfId')
```

Comment:
Hmm, maybe holding off on this for now is okay? This is from the Javascript side of the application; seems to touch a lot of code… I'm also okay reverting this change if you think that'd be preferable?